### ISSUE TITLE
Submission

### ISSUE THREAD

**Issue by ekilah** [ekilah/codepathtodo#1]
@codepath @codepathreview My app is complete, please review. Thanks!

**Comment by nidhi1608** [comment]
Monroe,

Looks good! This was intended in part to give you an introduction to the general rhythm of this course. The course is entirely project based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer to learn all practical Android development and best practices as quickly as possible. We also do a personalized code review for each submitted project once the bootcamp starts.

The next step is to continue working on [extensions to your todo app](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo) and **schedule a short 5-10 minute phone conversation** [here](https://www.google.com/calendar/selfsched?sstoken=UURvT01vbzBza1pnfGRlZmF1bHR8NzVjYzBkN2EyZTRiOGRmYjZlZjhmMjA2N2IyMGMwYWU). Navigate to [January 19th and dates onward](https://www.google.com/calendar/selfsched?sstoken=UURvT01vbzBza1pnfGRlZmF1bHR8NzVjYzBkN2EyZTRiOGRmYjZlZjhmMjA2N2IyMGMwYWU) and choose a 15-minute slot. Let us know if none of those times work.

Please make sure to **include the best number** to reach you at in the scheduled event? Look forward to chatting soon!

**Comment by nesquena** [comment]
Also can you verify you are eligible for this course based on the [requirements](http://courses.codepath.com/snippets/intro_to_android/thanks_for_applying#heading-requirements) listed there. Reviewing your linkedin, I don't think you have 2 full years of professional software development experience?

**Comment by ekilah** [comment]
Before I schedule a phone call, I'll reply to your second comment.

I have worked in software both during school and out of school, but not for a full 2 years since graduation. However, I work for Zynga, which has slots reserved for this event. Zynga's learning department told us a CS degree OR 2+ yrs of software work were good enough, not both. Is this still going to be an issue?

**Comment by nesquena** [comment]
Yeah sorry if there's any confusion in the messaging. We absolutely have spots for Zynga in the program. I will add you to the wait list now and if there are spots remaining for Zynga then happy to let you attend as a Zynga spot. Will keep in contact with you via email.

**Comment by ekilah** [comment]
Updated README and GIF with some new additions!

Not sure if I'm wasting my time asking for review again or not, but I would hope I could still be in line for a regular spot given my efforts to update the demo app with new features to prove my competency. I'm not sure that a few months of experience difference will matter, and I'd really like to be a part of this course.

Thanks.

**Comment by nesquena** [comment]
Glad to see you got in the datepicker fragment, the custom list adapter, etc. I don't think you are wasting your time. I hope you found the exercise useful so far. Keep in mind we've have over 350 total applications for the Android bootcamp, and every serious applicant is expected to include multiple extensions. For anyone that doesn't make it in we have a self-study observer program where they have full access to videos, resources, assignments and can follow along. We only have 25 engineers in each class so we have to select based on a variety of factors and one of those inevitably has to be experience. Keep in mind the bootcamp is targeted at senior engineers, the average work experience now across a cohort tends to be 5-7+ years of professional experience. 

That all said, appreciate your interest and it's still likely you will be able to fill one of the 5 zynga slots we've given them. Either way, you'll have access to the materials too.

**Comment by nesquena** [comment]
That all said, I'd encourage you when you have a chance next to tackle SQLite persistence next either directly through SQLiteOpenHelper or with an ORM (links in the prework). Makes for a good addition and you should find it really useful hopefully.

**Comment by ekilah** [comment]
Ah, makes sense. I appreciate the clarification! I will definitely follow up with the other options if necessary! 

And yeah, I'm gonna look into that next I think! Thanks for the help.
